packages/dx-react-grid/docs/guides/virtual-scrolling.md

```diff
@@ -38,6 +38,14 @@ If the Grid should have the same size as the container element, set the `Virtual
 
 .embedded-demo({ "path": "grid-virtual-scrolling/stretching-to-parent-element", "showThemeSelector": true })
 
+## Scroll to Row
+
+The virtual table can be scrolled programmatically, through the `scrollToRow` methods. To call these, you should use `VirtualTable` plugin's ref property. The following demo demonstrates how scroll table to new and added rows:
+
+.embedded-demo({ "path": "grid-virtual-scrolling/scroll-to-row", "showThemeSelector": true })
+
+Note that the virtual table with Lazy Loading doesn't allow to scroll to row's id.
```

Comment:
```suggestion
NOTE: Scrolling to a row does not work with lazy loading. This feature requires the IDs of all rows, but in lazy loading mode, the Grid loads rows in parts.
```

Reply:
Please make the phrase "lazy loading" a link to [this article](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/lazy-loading/#react-grid---virtual-scrolling-with-remote-data-lazy-loading).